src/modm/ui/display/monochrome_graphic_display.hpp

```diff
@@ -76,7 +76,28 @@ class MonochromeGraphicDisplay : public GraphicDisplay
 	clear() final;
 
 protected:
-	uint8_t buffer[BufferWidth][BufferHeight];
+	uint8_t buffer[BufferHeight][BufferWidth];
+
+	inline bool
+	xOnScreen(const int16_t x) const
+	{
+		return x >= 0 and x < int16_t(Width);
+	}
+	inline bool
+	yOnScreen(const int16_t y) const
+	{
+		return y >= 0 and y < int16_t(Height);
+	}
+	inline bool
+	xValidBuffer(const std::size_t x) const
+	{
+		return x >= 0 and x < BufferWidth;
+	}
+	inline bool
+	yValidBuffer(const std::size_t y) const
+	{
+		return y >= 0 and y < BufferHeight;
+	}
```

Comment:
Inline police ;-p

Reply:
Honestly??? But is not a template methode and should really be inlined! (Is still don't get it, propably)